dvc/remote/s3.py

```diff
@@ -301,3 +306,46 @@ def walk_files(self, path_info, max_items=None):
                 continue
 
             yield path_info.replace(path=fname)
+
+    @cached_property
+    def transfer_config(self):
+        from boto3.s3.transfer import TransferConfig
+
+        return TransferConfig()
+
```

Comment:
Not sure why we need this one.

Reply:
To access max threads number. The other way would be to make constant, but how can we be sure that amazon will not change defaults at some point?

Reply:
oh, right. Sorry, didn't notice that it is used. Makes sense now.